contracts/governance/script/DeployGovernance.s.sol

```diff
@@ -0,0 +1,81 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.19;
+
+import "forge-std/Script.sol";
+import "forge-std/console.sol";
+import "forge-std/console2.sol";
+import "@openzeppelin/token/ERC20/IERC20.sol";
+import "@openzeppelin/proxy/ERC1967/ERC1967Proxy.sol";
+
+import "../src/ZRXWrappedToken.sol";
+import "../src/ZeroExVotes.sol";
+import "../src/ZeroExTimelock.sol";
+import "../src/ZeroExProtocolGovernor.sol";
+import "../src/ZeroExTreasuryGovernor.sol";
+
+contract Deploy is Script {
+    address internal constant DEPLOYER = 0xd56716E3cC851bAF569795351917df73F8762A7F;
```

Comment:
I assume this is on a hardware wallet you own or something?

Reply:
Yeah again ignore all key and wallet references as this was my local testing.

Reply:
The deployer for everything except the vanity contracts will be 0xEf37aD2BACD70119F141140f7B5E46Cd53a65fc4